solidity/migrations/2_sausage_migration.js

```diff
@@ -1,20 +1,143 @@
+      await auctionInstance.startAuction(1000, { from: managerAccount }) //auction has 1000 seconds
+
+      let originalBid = 1
+      await auctionInstance.bid({ from: managerAccount, value: originalBid }) //note, value is 10 wei, or 10^-18 ether
+
+      let managerAddr = await auctionInstance.manager.call()
+      let erc721ContractAddr = await auctionInstance.erc721TokenAddr.call()
+
+      let firstBid = await auctionInstance.latestBid.call()
+      let firstBidder = await auctionInstance.latestBidder.call()
+      let secondsAuction = await auctionInstance.auctionSecondsLeft.call()
+      let uniqueID = await auctionInstance.uniqueID.call()
+      let sausageType = await auctionInstance.sausageName.call()
+
+      let secondsNow = Math.round(new Date().getTime() / 1000)
+      let secondsLeft = secondsAuction - secondsNow
+      console.log(`************* MANAGER ADDRESS      : ${managerAddr} `)
+      console.log(`************* ERC721 TOKEN ADDRESS : ${erc721ContractAddr} `)
+      console.log('\n')
+      console.log(
+        `************* MIGRATIONS SUCCESSFULLY LAUNCHED AUCTION. SEE BELOW *************`
+      )
+      console.log('\n')
+      console.log(`************* AUCTION ADDRESS      : ${auctionAddress} `)
+      console.log(`************* FIRST BID            : ${firstBid} `)
+      console.log(`************* FIRST BIDDER         : ${firstBidder} `)
+      console.log(`************* AUCTION SECONDS LEFT : ${secondsLeft} `)
+      console.log(`************* UNIQUE SAUSAGE ID    : ${uniqueID} `)
+      console.log(`************* SAUSAGE TYPE         : ${sausageType} `)
+
+      console.log(
+        `************* BELOW WE RUN A SCRIPT TO MAKE 100 BIDS SO THE SUBGRAPH HAS SOMETHING TO TRACK! *************`
+      )
 
-//   module.exports = function(deployer) {
-//     deployer.deploy(SausageTokens, "SausageTokens", "Bratwurst").then(() => {
-//      deployer.deploy(Auction, "0x2259fefc212cc383004e1f4ec280230231b8c342", 1234, "Bratwurst")
-//     });
-// };
+      // note, each bid adds 10 seconds to the auction length. so 1000 seconds will be added
+      let bidderCounter = originalBid
+      let bidderNum = 1
+      for (let i = 1; i < 51; i++) {
+        let bidder_1_value = bidderCounter + 1
+        let bidder_2_value = bidder_1_value + 1
+        bidderCounter = bidder_2_value
+        console.log(`BID ${bidderNum}: ${bidder_1_value}`)
+        await auctionInstance.bid({ from: accounts[1], value: bidder_1_value }) //note, value is 10 wei, or 10^-18 ether
+
+        console.log(`BID ${bidderNum + 1}: ${bidder_2_value}`)
+        await auctionInstance.bid({ from: accounts[2], value: bidder_2_value }) //note, value is 10 wei, or 10^-18 ether
+
+        bidderNum += 2
+      }
+
+      //note, we will not bid here, so the token winner ends up being 0x00000000...
+      console.log(
+        `************* NEXT WE WILL MAKE AN AUCTION THAT ENDS RIGHT AWAY, TO GET event LogAcountClosed() TO TEST THE GRAPH *************`
```

Comment:
Test 2 doesnt work. lets also put this in a script. the script should actually make and end like 100 auctions, so we can get 100 closeAuctionEvents() returned to use the graph nicely!